driver/src/main/com/mongodb/reactivestreams/client/MongoCollection.java

```diff
@@ -1054,6 +1053,66 @@
      */
     Publisher<UpdateResult> updateOne(ClientSession clientSession, Bson filter, Bson update, UpdateOptions options);
 
+    /**
+     * Update a single document in the collection according to the specified arguments.
+     *
+     * <p>Note: Supports retryable writes on MongoDB server versions 3.6 or higher when the retryWrites setting is enabled.</p>
+     * @param filter a document describing the query filter, which may not be null.
+     * @param update a pipeline describing the update, which may not be null.
+     * @return a publisher with a single element the UpdateResult
+     * @since 1.12
+     * @mongodb.server.release 4.2
+     * @mongodb.driver.manual tutorial/modify-documents/ Updates
+     * @mongodb.driver.manual reference/operator/update/ Update Operators
+     */
+    Publisher<UpdateResult>  updateOne(Bson filter, List<? extends Bson> update);
```

Comment:
extra whitespace before method name in a bunch of places in this file

Reply:
Thanks - fixed